src/manageiq_client/api.py

```diff
@@ -322,7 +342,7 @@ class Entity(object):
     SUBCOLLECTIONS = dict(
         service_catalogs={"service_templates"},
         roles={"features"},
-        providers={"tags"},
+        providers={"tags", "custom_attributes"},
```

Comment:
I'm not sure what this is adding since it is already possible to query and create/update/delete custom attributes, but if added vms also support custom attributes (line #353)